src/frontend/src/pages/AdminToolsPage/RecruitmentConfig/ApplicationLinkTable.tsx

```diff
@@ -0,0 +1,44 @@
+import { TableCell, TableRow, Box, Typography } from '@mui/material';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { useCurrentOrganization } from '../../../hooks/organizations.hooks';
+import AdminToolTable from '../AdminToolTable';
+import React from 'react';
+
+const ApplicationLinkTable: React.FC = () => {
+  const {
+    data: organization,
+    isLoading: organizationIsLoading,
+    isError: organizationIsError,
+    error: organizationError
+  } = useCurrentOrganization();
+
+  if (!organization || organizationIsLoading) {
+    return <LoadingIndicator />;
+  }
+  if (organizationIsError) {
+    return <ErrorPage message={organizationError.message} />;
+  }
```

Comment:
Instead of doing all this in the sidebar file, do this in AppAuthenticated and then pass in organization as a prop into sidebar